src/NuGet.Clients/NuGet.Credentials/SecurePluginCredentialProvider.cs

```diff
@@ -199,11 +200,9 @@ private static CredentialResponse GetAuthenticationCredentialsResponseToCredenti
             CredentialResponse taskResponse;
             if (credentialResponse.IsValid())
             {
-                ICredentials result = new NetworkCredential(credentialResponse.Username, credentialResponse.Password);
-                if (credentialResponse.AuthenticationTypes != null)
-                {
-                    result = new AuthTypeFilteredCredentials(result, credentialResponse.AuthenticationTypes);
-                }
+                ICredentials result = new AuthTypeFilteredCredentials(
```

Comment:
Likewise...I think this is an unnecesarry change as it has no functional impact.